service/controller/v1/prometheus/prometheus.go

```diff
@@ -349,6 +352,11 @@ var (
 	IngressWhitelistRegexp = relabel.MustNewRegexp(`(kube-system;nginx-ingress-controller)`)
 )
 
+type Config struct {
```

Comment:
Introduced this to store both configuration options which are passed to GetScrapeConfigs -- the certificate directory and the provider.

Reply:
Added